package.json

```diff
@@ -142,6 +142,7 @@
     "mochawesome-merge": "^4.2.1",
     "pluralize": "^8.0.0",
     "prettier": "^2.5.1",
+    "reselect": "^4.1.6",
```

Comment:
`reselect` as peer dep breaks new tests. Now dev dep. I'll remove it with the new SDK version, I promise